### PR TITLE
Home CSS

### PR DESCRIPTION
**Pull request by bwormser**
Added home css styling. Needs to add correct font but pull request so home javascript can be worked on.

### CODE REVIEW

File: src/static/home.html [hunk 1]
```diff
@@ -6,78 +6,84 @@
...
+                    </div>
                 </div>
             </div>
-
-            <!-- Sign up with Email button also opens up signup modal above -->
-            <div>
-                <label for="signup" class="signup-btn">Sign Up with Email!</label>
+        </div>
+        <main>
+            <div class="home-main">
```
Comment (mpolimen) [inline src/static/home.html:72]: div with class="home-main" could just use the main tag that already encompasses all of the same content

**Comment by mpolimen** [comment]
Bumping the font code

@import url('https://fonts.googleapis.com/css2?family=Barlow:ital@1&display=swap%27);
font-family: 'Barlow', sans-serif;
@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap%27);
font-family: 'Roboto', sans-serif;

File: src/static/home.html (hunk 1 above)
Comment (bwormser) [reply src/static/home.html]: Yep fixed that

**Comment by roccopolimen** [comment]
We should NOT be doing CSS with pixels. It'll look terrible on some screens and fine on others. try to use % mostly. Fine for now i guess because this pull request is a blocker.

**Pull request merged**